doc/source/how-to/compatibility.rst

```diff
@@ -0,0 +1,77 @@
+Ansys products compatibility
+============================
+As the different PyAnsys libraries evolve, backwards and forwards compatibility
+issues may rise. Some of the most common cases are:
+
+* An Ansys product has implemented certain features in its new server version
+  which are not available in previous server versions. This causes backwards
+  incompatibility issues.
+* New server versions have suppressed support for certain operations after a
+  a given version. This causes forwards incompatibility issues.
+
+Though there are different ways to handle these issues, some of the PyAnsys libraries,
+such as `PyMAPDL <https://github.com/pyansys/pymapdl>`_ and
+`PyDPF-Core <https://github.com/pyansys/pydpf-core>`_, have started to handle them in
+the same way. In order to homogenize implementations throughout the libraries,
+it is recommended to follow their approach.
+
+``check_version.py`` module approach
+------------------------------------
+As mentioned before, PyAnsys libraries have started to implement *version checking*
+modules which allow to determine whether the Ansys product server you are connecting
+to provides support for certain operations. Examples of this module may be found for
+the following libraries:
+
+* `PyMAPDL check_version.py module <https://github.com/pyansys/pymapdl/blob/main/src/ansys/mapdl/core/check_version.py>`_
+* `PyDPF-Core check_version.py module <https://github.com/pyansys/pydpf-core/blob/master/ansys/dpf/core/check_version.py>`_
+
+One of the easiest ways to keep track of the versions supported is setting up a
+**minimum version** data structure, in which forwards compatibility is ensured.
+Server versions prior to the minimum version do not have access to this
+functionality. In the case of the `PyMAPDL check_version.py module`_, this is the
+``VERSIONS_MAP`` structure.
+
+Most Ansys products provide forwards compatibility, meaning that whatever feature
+introduced in a previous version, is typically supported in upcoming versions
+as well. In case that a feature was decided to be suppressed, this would lead to
+backwards compatibility issues. The same would happen with the PyAnsys servers wrapping
+the Ansys products. Thus, for this case, it would also be necessary to create a
+similar data structure, but in this case for a **maximum version**. Though there are
+no implementations yet of this feature, the way it should work would be analogous to
+the way the **minimum version** mechanism does.
```

Comment:
```suggestion
Most Ansys products provide forward compatibility, meaning that features
introduced in an earlier version are also supported in later versions. Suppressing
a feature would lead to a backward compatibility issue.

The same happen with the PyAnsys servers wrapping
the Ansys products. Thus, for this case, it would also be necessary to create a
similar data structure, but in this case for a **maximum version**. Though there are
no implementations yet of this feature, the way it should work would be analogous to
the way the **minimum version** mechanism does.
```